nearby/nearby.proto

```diff
@@ -39,4 +39,6 @@ message NearbyOutput {
 message PlaceRestriction {
   uint32 place_id = 1;
   uint32 restriction_id = 2;
+  Coordinate place = 3;
```

Comment:
place is ambiguous, please rename to ```place_coord``` or something specific

Reply:
needs to match the output

`data.place_restriction.place.latitude`

```
{
  "data": {
    "place_restriction": [
      {
        "place_id": 912,
        "restriction_id": 6,
        "place": {
          "latitude": null,
          "longitude": null
        }
      },
      {
        "place_id": 991,
        "restriction_id": 6,
        "place": {
          "latitude": null,
          "longitude": null
        }
      },
      {
        "place_id": 993,
        "restriction_id": 6,
        "place": {
          "latitude": null,
          "longitude": null
        }
      },
      {
        "place_id": 994,
        "restriction_id": 6,
        "place": {
          "latitude": null,
          "longitude": null
        }
      },
      {
        "place_id": 995,
        "restriction_id": 6,
        "place": {
          "latitude": null,
          "longitude": null
        }
      },
      {
        "place_id": 996,
        "restriction_id": 6,
        "place": {
          "latitude": null,
          "longitude": null
        }
      },
      {
        "place_id": 997,
        "restriction_id": 6,
        "place": {
          "latitude": null,
          "longitude": null
        }
      },
      {
        "place_id": 1000,
        "restriction_id": 6,
        "place": {
          "latitude": null,
          "longitude": null
        }
      },
      {
        "place_id": 912,
        "restriction_id": 8,
        "place": {
          "latitude": null,
          "longitude": null
        }
      },
      {
        "place_id": 991,
        "restriction_id": 8,
        "place": {
          "latitude": null,
          "longitude": null
        }
      },
      {
        "place_id": 993,
        "restriction_id": 8,
        "place": {
          "latitude": null,
          "longitude": null
        }
      },
      {
        "place_id": 994,
        "restriction_id": 8,
        "place": {
          "latitude": null,
          "longitude": null
        }
      },
      {
        "place_id": 995,
        "restriction_id": 8,
        "place": {
          "latitude": null,
          "longitude": null
        }
      },
      {
        "place_id": 996,
        "restriction_id": 8,
        "place": {
          "latitude": null,
          "longitude": null
        }
      },
      {
        "place_id": 997,
        "restriction_id": 8,
        "place": {
          "latitude": null,
          "longitude": null
        }
      },
      {
        "place_id": 1000,
        "restriction_id": 8,
        "place": {
          "latitude": null,
          "longitude": null
        }
      },
      {
        "place_id": 1209,
        "restriction_id": 8,
        "place": {
          "latitude": null,
          "longitude": null
        }
      }
    ],
    "user_by_pk": {
      "user_restrictions": [
        {
          "restriction_id": 6
        },
        {
          "restriction_id": 8
        }
      ],
      "id": "NmHOHf8TWYYUPn2Oc7YXnJxXjnI2"
    }
  }
}
```